frontend/src/js/controllers/authCtrl.js

```diff
@@ -55,6 +55,22 @@
             vm.wrnMsg = {};
         }
 
+        vm.passwordCheck = function() {
+            var parameters = {};
+            parameters.data = {
+                "password1": vm.regUser.password,
+                "password2": vm.regUser.confirm
+            }
+            if (parameters.data.password1 !== "" && parameters.data.password1 === parameters.data.password2)
+            {
+                return false;
```

Comment:
@spyshiv Actually it is because we are showing the message on the frontend as `Passwords do not match`. So, If I change this to true then we have to alter the message their as `Passwords Match`. What should I follow ?